src/hair/hairStyle.css

```diff
@@ -0,0 +1,144 @@
+@import url('https://fonts.googleapis.com/css?family=Josefin+Sans:400,700&display=swap');
+
+body {
+  height: 100vh;
+  width: 100vw;
+  background-image: url(./img/mainbackgroundimg.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  /* how do i know when to do vh vs %? */
```

Comment:
you don't. you do what works and what looks good. lol. height is a finicky property. there may be an actual rule about when to use either but I have researched. if you do, please let me and the class know!